src/libs/Growl.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const growlRef = React.createRef();
+
+/**
+ * Show the growl notification
+ *
+ * @param {String} bodyText
+ * @param {String} type
+ * @param {Number} duration
+*/
+function show(bodyText, type, duration) {
+    growlRef.current.show(bodyText, type, duration);
```

Comment:
Is this a safe reference? Should we instead do:

```suggestion
    if (_.isFunction(lodashGet(growlRef, 'current.show'))) {
        growlRef.current.show(bodyText, type, duration);
    }
```

Reply:
Its safe as `<Expensify />` component will be rendered before anything and ref will always be present.